rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/BazelBuildFileView.java

```diff
@@ -94,7 +94,7 @@ private String assembleGoImportPath(boolean isCloud, String protoPkg, String goP
     goPkg = goPkg.replaceFirst("cloud\\/", "");
 
     String goImport = "";
-    if (isCloud) {
+    if (isCloud || protoPkg.contains("cloud")) {
```

Comment:
This looks weird. `isCloud` should define if it is a cloud or no. Here it does it partially, and it can be overriden by protoPkg value (which also an argument to this function).

Please make sure that isCloud completely defines the cloud thing. (i.e. it an be as straightforward as moving `protoPkg.contains("cloud")` from here to the place where `assembleGoImportPath` is defined.

Reply:
Ah, good call. I did just that! Done.